lib/symbol-store.coffee

```diff
@@ -174,6 +174,7 @@ class SymbolStore
 
     for bufferRow in [bufferRange.start.row..bufferRange.end.row] by 1
       tokenizedLine = tokenizedLines[bufferRow]
```

Comment:
I suspect this could involve some kind of issue with event order plus split panes, but it's still unclear. We subscribe to the text buffer with `preemptDidChange` in `TokenizedBuffer`, which should ensure that we always get these events first.